### PR TITLE
Bumped Ruby version and added SimplePay to list of companies

### PR DESCRIPTION
**Pull request by orangethunder**
Hi! I was at your last meetup on Tuesday.

I'm adding my company here as suggested, and since we're hiring.

I bumped the ruby version required by the Gemfile to 2.4.4, since rvm gives warnings about security when installing 2.4.0.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.4.0'
+ruby '2.4.4'
```
Comment (Drenmi) [inline Gemfile]: Let's just go straight to 2.5.1? 🙂

**Comment by Drenmi** [comment]
You'll also need to update `.ruby-version` and `.travis.yml`

**Comment by orangethunder** [comment]
Updated to Ruby 2.5.1 now. All specs pass, however I haven't done any manual testing of the site. There are no breaking changes I'm aware of between 2.4 and 2.5.

**Comment by hanmd82** [comment]
I can do a deployment to staging server tonight to test it out :)

**Comment by orangethunder** [comment]
@hanmd82 I decided to do a Rails 5 upgrade while I was at it. Specs are passing but I just want to clean up the code a bit before committing. If you want, you can wait a bit before doing the tests on staging :)

**Comment by hanmd82** [comment]
tested on staging (https://rubysg-staging.herokuapp.com/companies) - works with no regression.. LGTM 👍

**Comment by hanmd82** [comment]
deployed to production.. thanks @orangethunder 🙇

**Pull request merged**